firstpartydata/tests/applyfpd/valid-fpd-all-bidder-app.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
This is related to some of the other comments on UT vs integration tests and the JSON framework, but a lot of these tests present themselves as integration tests on the surface but appear to actually be unit tests. The corresponding test that runs one of these JSON scripts in `first_party_data_test.go` only utilizes a piece of `inputRequestData` and verifies a piece of `outputRequestData`. To confirm this, I was able to reduce this file from 188 lines to 77 and still get it to pass. As mentioned earlier, it would probably be a good idea to test full requests and verify entire output requests in the exchange package and pare down these tests here to the bare minimum needed to test the function, whether that be through this JSON strategy or a typical go test.

Reply:
Agree, unused data removed